akka-docs/src/main/paradox/java/stream/stream-composition.md

```diff
@@ -35,7 +35,7 @@ hiding them behind a *shape* that looks like a `Source`, `Flow`, etc.
 
 One interesting example above is a `Flow` which is composed of a disconnected `Sink` and `Source`.
 This can be achieved by using the `fromSinkAndSource()` constructor method on `Flow` which takes the two parts as
-parameters.
+parameters. 
```

Comment:
Not a big deal here, but it'd be neat if we could have something like scalariform/scalafmt for our markdown to remove trailing whitespace :)